README.md

```diff
@@ -660,6 +661,36 @@ These are values that represent the current state of the downshift component.
 As a convenience, the `id` and `itemToString` props which you pass to
 `<Downshift />` are available here as well.
 
+## EventHandlers
+
+You can provide your own event handlers to Downshift which will be called before the default handlers.
+
+If you would like to prevent the default handler behavior in some cases, you can set the event's `preventDownshiftDefault` property to `false`:
+
+```javascript
+getInputProps({
+  onKeyDown(event) {
+    if (event.key === 'Enter') {
+      // Prevent Downshift's default 'Enter' behavior.
+      event.preventDownshiftDefault = false
+
+      // Implement my own behavior...
+    }
+  },
+})
+```
+
+If you would like to completely override Downshift's behavior for a handler, in favor of your own, you can bypass prop getters:
+
+```javascript
+let inputProps = {
+  ...getInputProps(customProps),
+  onKeyDown(event) {
+    // Implement my own behavior...
+  },
+}
+```
+
```

Comment:
Cool, thanks for the update. Could we make these examples more practical so folks who may be more beginner will understand. So like:

```javascript
<input {...getInputProps()} onKeyDown={event => { /* my own stuff */} />
```

And the same for the example above?